toasty/image.py

```diff
@@ -1205,7 +1214,7 @@ def save(
             if format in PIL_RGB_FORMATS and mode is None:
                 mode = ImageMode.RGB
             if mode is not None:
-                pil_image = pil_image.convert(mode.try_as_pil())
+                pil_image = pil_image.convert(mode.value)
```

Comment:
I'm a little wary of dropping the check here since there are ImageMode values that are not valid inputs to `Image.convert()`. We need to check for those at some point.